user-manual/preservation/preservation-planning.rst

```diff
@@ -178,10 +178,9 @@ Tools
 ^^^^^
 
 Archivematica acts as a wrapper for many open source tools used to carry out
-preservation actions. These include digital preservation specific tools like
-`FITS`_, used for extracting technical metadata from files, as well as tools for
-handling different file formats like `Inkscape`_, which is a design program used
-to handle vector images.
+preservation actions. These include digital preservation specific tools as well
+as tools for handling different file formats like `Inkscape`_, which is a design
+program used to handle vector images.
```

Comment:
I'm going to suggest a reword of these three lines so that we avoid mentioning any tool specifically:
"preservation actions. These include digital preservation specific tools (such as those to carry out file format identification and characterization) as well as generalist tools used for actions such as normalization."